pkg/cloudprovider/providers/gce/gce_alpha.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import "fmt"
+
+const (
+	allAlphaFeatures = "all"
+	noneAlphaFeature = "none"
+)
+
+// All supported alpha features
+var supportedAlphaFeatures = map[string]bool{}
+
+type AlphaFeatureGate struct {
+	features map[string]bool
+}
+
+func (af *AlphaFeatureGate) Enabled(key string) bool {
+	return af.features[key]
```

Comment:
Are you using the existence of the key or the actual boolean value to determine whether a feature is enabled?